packages/accounts-password/password_tests.js

```diff
@@ -1886,4 +1886,51 @@ if (Meteor.isServer) (function () {
     ]);
   });
 
+  Tinytest.addAsync(
+    'passwords - allow custom bcrypt rounds',
+    function (test, done) {
+      function getUserHashRounds(user) {
+        return Number(user.services.password.bcrypt.substring(4, 6));
```

Comment:
Let's use `parseInt` here to avoid interpreting leading 0s as octal.

Reply:
Done - and I've adjusted rounds extraction to use split string segments. Thanks!